Discord Mafia Bot .Net/Commands/Commands.cs

```diff
@@ -243,6 +243,7 @@ public class Game_Commands : ModuleBase
                             if (everyoneReady && game.Objects.Count > 4)
                             {
                                 await ReplyAsync("", false, new EmbedBuilder() { Title = "Game Start!", Color = Color.Green, Description = $"@everyone is ready! Starting up the game..." });
+                                IGuildChannel.signupchannel = Context.Channel;
```

Comment:
You need to set the SignupChannel to the Game object, if you add this commit to the other PRs (aka merge them all together) I'll fix this part. :)